techniques/system/common/1.0/cf-serverd.st

```diff
@@ -73,7 +73,7 @@ bundle server access_rules
         comment            => "Grant ${query_types} reporting query for the hub on the policy server",
         resource_type      => "query",
         report_data_select => rudder_data_select_policy_hub,
-        admit_ips          => { host2ip("${def.policy_server}"), @{sys.ip_addresses} }; # an enterprise policy server needs to be able to contact itself
+        admit_ips          => { host2ip("${server_info.policy_server}"), @{sys.ip_addresses} }; # an enterprise policy server needs to be able to contact itself
```

Comment:
policy file cf-serverd is loaded before file update.cf
are we sure that the agent knows how to resolve these variable in the right order ?

Reply:
I'm pretty sure it works since this variable is already used below, line 102